### PR TITLE
adding minor version of XrootD to the requirements (software-4137)

### PR DESCRIPTION
**Pull request by ddavila0**

### CODE REVIEW

File: rpm/xrootd-multiuser.spec [hunk 1]
```diff
@@ -12,22 +12,23 @@ URL: https://github.com/bbockelm/xrootd-multiuser
...
 
 BuildRoot: %(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
-BuildRequires: xrootd-server-libs >= 1:%{xrootd_current_major}.0.0-0
-BuildRequires: xrootd-server-libs <  1:%{xrootd_next_major}.0.0-0
-BuildRequires: xrootd-server-devel >= 1:%{xrootd_current_major}.0.0-0
-BuildRequires: xrootd-server-devel <  1:%{xrootd_next_major}.0.0-0
+BuildRequires: xrootd-server-libs >= 1:%{xrootd_current_major}
+BuildRequires: xrootd-server-libs <  1:%{xrootd_next_major}
+BuildRequires: xrootd-server-devel >= 1:%{xrootd_current_major}
+BuildRequires: xrootd-server-devel <  1:%{xrootd_next_major}
```
Comment (brianhlin) [inline rpm/xrootd-multiuser.spec:22]:
```suggestion
BuildRequires: xrootd-server-libs = 1:%{xrootd_current_major}
BuildRequires: xrootd-server-devel = 1:%{xrootd_current_major}
```

Can't we simplify the 4 requirements down to 2?

File: rpm/xrootd-multiuser.spec [hunk 2]
```diff
@@ -12,22 +12,23 @@ URL: https://github.com/bbockelm/xrootd-multiuser
...
 BuildRequires: gcc-c++
 BuildRequires: libcap-devel
 %{?systemd_requires}
 # For %{_unitdir} macro
 BuildRequires: systemd
 
-Requires: xrootd-server >= 1:%{xrootd_current_major}.0.0-0
-Requires: xrootd-server <  1:%{xrootd_next_major}.0.0-0
+Requires: xrootd-server >= 1:%{xrootd_current_major}.%{xrootd_current_minor}
+Requires: xrootd-server <  1:%{xrootd_next_major}.0.0-1
```
Comment (brianhlin) [inline rpm/xrootd-multiuser.spec:31]: Do we expect pre-release builds of the next major version to work with plugins built against the current major version?

**Comment by brianhlin** [comment]
@ddavila0 If you make any changes for my above comments please also update your PRs to the other plugins

File: rpm/xrootd-multiuser.spec (hunk 2 above)
Comment (ddavila0) [reply rpm/xrootd-multiuser.spec:31]: yeah, that's why we have  ".0.0-1" :   "0.0-rc < 0.0-1"

File: rpm/xrootd-multiuser.spec (hunk 1 above)
Comment (ddavila0) [reply rpm/xrootd-multiuser.spec:22]: Nop, it looks like if you use "=" you have to specify the full version major.minor.x and unless we want to change the spec every time we have a new version of xrootd, I don't think we can do this change.

**Pull request merged**